### ISSUE TITLE
[DEPRECATED] DO NOT USE THIS LIBRARY ANYMORE - Read for full details

### ISSUE THREAD

**Issue by BrodaNoel** [archriss/react-native-snap-carousel#959]
This was a lovely library, but it wasn't updated during more than 2 years (I just hope COVID didn't have anything to do there - Even the Github user who created this library seems that was deleted (`Benoît Delmaire`)).

We are all migrating to https://www.npmjs.com/package/react-native-reanimated-carousel

<img width="400" alt="Screen Shot 2022-10-18 at 17 22 13" src="https://user-images.githubusercontent.com/1837969/196536080-b3d63431-77a3-4a67-8a7e-5e41fd328920.png">

It is important that you migrate out of this library as soon as possible, in order to stop promoting it.

<img width="214" alt="Screen Shot 2022-10-18 at 17 23 33" src="https://user-images.githubusercontent.com/1837969/196536331-be6340c4-d17a-4a0d-ba7d-3186c74d255e.png">

See you in the other side.

**Comment by BrodaNoel** [comment]
And please stop creating other issues in this library so this one keeps on the top.

**Comment by mvrahas** [comment]
Thank you

**Comment by mm-webx** [comment]
👍

**Comment by ciberbob-007** [comment]
Guys i already create a fork, because i think something happened with this developer... try using my solutions on

https://www.npmjs.com/package/react-native-new-snap-carousel

hope this help

**Comment by BrodaNoel** [comment]
> Guys i already create a fork, because i think something happened with this developer... try using my solutions on
> 
> 
> 
> https://www.npmjs.com/package/react-native-new-snap-carousel
> 
> 
> 
> hope this help 

Why not just https://www.npmjs.com/package/react-native-reanimated-carousel ? We are all migrating there.

**Comment by ciberbob-007** [comment]
That is a decision for every developer.. some of them just want to maintain legacy code without so much effort, others would like to implement a new library 🤷🏻‍♂️

**Comment by Tieantono** [comment]
> Guys i already create a fork, because i think something happened with this developer... try using my solutions on
> 
> https://www.npmjs.com/package/react-native-new-snap-carousel
> 
> hope this help

Thanks, this helps us who in need to migrate to the newest Expo and RN versions but also need more time to migrate to `react-native-reanimated-carousel`.

**Comment by mhkhung** [comment]
> That is a decision for every developer.. some of them just want to maintain legacy code without so much effort, others would like to implement a new library 🤷🏻‍♂️

I can't find the fork on github? npmjs is still pointing to this repo..

**Comment by ChronSyn** [comment]
> I can't find the fork on github? npmjs is still pointing to this repo..

NPM will still point towards this repo unless the owner of the NPM project publishes an update (or transfers it to someone else and then the new owner publishes an update).

You can `npm install https://github.com/<username>/<repository>` and it'll install from a github-sourced repository instead of the NPM repository.

**Comment by mkyy** [comment]
I've lost a plenty of time with this.

**Comment by aarshad-devani** [comment]
Forked Repository - https://github.com/ciberbob-007/react-native-new-snap-carousel
Credits - @ciberbob-007

**Comment by zeusstl** [comment]
Commenting to bump this issue to the top in the issues view.

**Comment by leegun** [comment]
> Guys i already create a fork, because i think something happened with this developer... try using my solutions on
> 
> https://www.npmjs.com/package/react-native-new-snap-carousel
> 
> hope this help

there are no @type repository
how install react-native-new-snap-carousel??

Old snap carousel has @type repository
https://www.npmjs.com/package/@types/react-native-snap-carousel

**Comment by leegun** [comment]
I solved it by writing the following in package.json.

```
"react-native-snap-carousel": "https://github.com/ciberbob-007/react-native-new-snap-carousel",
```

**Comment by danidaryaweesh** [comment]
@BrodaNoel I have tried to implement the new library you are suggesting, but I am unable to get it working with my current code due to some missing UI features. I am not finding the following props in the new library Parallax mode:

           itemWidth
            sliderWidth
            itemHeight
            sliderHeight
            activeSlideAlignment
            inactiveSlideScale
            inactiveSlideOpacity
            containerCustomStyle
            contentContainerCustomStyle
            
            
Any help or guidance will be very appreciated. Thanks!

**Comment by BrodaNoel** [comment]
> @BrodaNoel I have tried to implement the new library you are suggesting, but I am unable to get it working with my current code due to some missing UI features. I am not finding the following props in the new library Parallax mode:
> 
> 
> 
>            itemWidth
> 
>             sliderWidth
> 
>             itemHeight
> 
>             sliderHeight
> 
>             activeSlideAlignment
> 
>             inactiveSlideScale
> 
>             inactiveSlideOpacity
> 
>             containerCustomStyle
> 
>             contentContainerCustomStyle
> 
>             
> 
>             
> 
> Any help or guidance will be very appreciated. Thanks!

Why are you asking this here? Ask it in the other library

**Comment by danidaryaweesh** [comment]
@BrodaNoel You're right I'm sorry, I should have posted that in the other library. I guess that I just thought that people who have migrated might have a better idea.

**Comment by ciberbob-007** [comment]
> @BrodaNoel I have tried to implement the new library you are suggesting, but I am unable to get it working with my current code due to some missing UI features. I am not finding the following props in the new library Parallax mode:
> 
> ```
>        itemWidth
>         sliderWidth
>         itemHeight
>         sliderHeight
>         activeSlideAlignment
>         inactiveSlideScale
>         inactiveSlideOpacity
>         containerCustomStyle
>         contentContainerCustomStyle
> ```
> 
> Any help or guidance will be very appreciated. Thanks!

I will try to do support to this library as much as possible, please ask in the fork repo on https://github.com/ciberbob-007/react-native-new-snap-carousel

**Comment by MnariMohamed** [comment]
i don't understand how is this still getting 122K weekly downloads, oh! maybe because of people like me who figures out that it is deprecated after starting the implementation.
